cmd/sriov/main.go

```diff
@@ -102,14 +102,24 @@ func cmdAdd(args *skel.CmdArgs) error {
 	}}
 
 	if !netConf.DPDKMode {
-		macAddr, err = sm.SetupVF(netConf, args.IfName, netns)
+		err = sm.SetupVF(netConf, args.IfName, netns)
 
 		if err != nil {
 			return fmt.Errorf("failed to set up pod interface %q from the device %q: %v", args.IfName, netConf.Master, err)
 		}
 		result.Interfaces[0].Mac = macAddr
 	}
 
+	// add the mac address we report to the CNI call return object
+	// if the device is on kernel mode we report that one back
+	// if not we check the administrative mac address on the PF
+	// in case that one is zero we will not report it
```

Comment:
nit: if it is set and is not zero, report  it.

Reply:
thanks :)